.github/dependabot.yml

```diff
@@ -4,11 +4,23 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    groups:
+      # Group all terraform-plugin-(go|sdk|framework|testing) dependencies together
+      "terraform-plugin":
+        patterns:
+          - "github.com/hashicorp/terraform-plugin-*"
```

Comment:
This repo only uses plugin-go and plugin-log, which rarely are released at the same time 🤔 
```suggestion
```